lizard_auth_server/models.py

```diff
@@ -317,6 +317,13 @@ def has_access(self, portal):
             return True
         return self.portals.filter(pk=portal.pk).exists()
 
+    @staticmethod
+    def has_profile(user, portal):
```

Comment:
Ook zou ik het misschien eerder als een losse functie dan als een `@staticmethod` op UserProfile doen. Misschien is het hier wel makkelijk, hoor, maar initieel vond ik het er wat raar uitzien.